compiler/rustc_expand/src/mbe/quoted.rs

```diff
@@ -62,7 +62,10 @@ pub(super) fn parse(
         match tree {
             TokenTree::MetaVar(start_sp, ident) if parsing_patterns => {
                 let span = match trees.next() {
-                    Some(&tokenstream::TokenTree::Token(Token { kind: token::Colon, span }, _)) => {
+                    Some(&tokenstream::TokenTree::Token(
+                        Token { kind: token::Colon, span: colon_sp },
```

Comment:
```suggestion
                        Token { kind: token::Colon, span: colon_span },
```
FWIW, I mildly hate the `sp` abbreviation, `span` is already short.

Reply:
lol, I originaly had `colon_span` and then changed it to `colon_sp` because of other `sp` variables nearby. I'll change it back.